### PR TITLE
Fix estimator predict performance

### LINKED ISSUE #1813: Predicting is rather slow
```sql
SELECT * FROM titanic.train TO TRAIN DNNClassifier 
  WITH model.hidden_units=[200, 200，200],optimizer.learning_rate=0.01
  LABEL survived
  INTO sqlflow_models.titanic_model;
```
```sql
SELECT * FROM titanic.test TO PREDICT titanic.predict.survived 
  USING sqlflow_models.titanic_model;
```
On my MacBook Pro, The `PREDICT` statement spent about ten minutes, which is too slow because there're only 418 samples in the `titanic.test` dataset.

### PR DESCRIPTION
**Pull request by typhoonzero**
Fix https://github.com/sql-machine-learning/sqlflow/issues/1813

### CODE REVIEW

File: pkg/sql/codegen/tensorflow/codegen.go [hunk 1]
```diff
@@ -308,6 +308,7 @@ func categorizeAttributes(trainStmt *ir.TrainStmt) (trainParams, validateParams,
 func deriveFeatureColumnCode(trainStmt *ir.TrainStmt) (featureColumnsCode []string, fieldDescs []*ir.FieldDesc, err error) {
 	perTargetFeatureColumnsCode := []string{}
 	for target, fcList := range trainStmt.Features {
+		perTargetFeatureColumnsCode = []string{}
```
Comment (tonyyang-svail) [inline pkg/sql/codegen/tensorflow/codegen.go]: Was this a bug? if so, how about removing line 309 and change this line to `:=`?

**Review by tonyyang-svail — DISMISSED** [review]

**Review by tonyyang-svail — APPROVED** [review]
Excellent!

**Review by tonyyang-svail — APPROVED** [review]
LGTM.

@typhoonzero FYI, I resolved the conflict introduced in https://github.com/sql-machine-learning/sqlflow/pull/1836 by removing the fast predict.

**Comment by typhoonzero** [comment]
@tonyyang-svail Thanks!

**Pull request merged**